tutorials/tutorial2.ipynb

```diff
@@ -49,7 +49,7 @@
     }
    ],
    "source": [
-    "%env atacworks=/ntadimeti/AtacWorks"
```

Comment:
Mention that this path should be replaced with the local path where the user has cloned AtacWorks.

Reply:
I will delete this path and replace it with `<path-to-atacworks>` after I run it.